.azure-pipelines/common2/steps/run-mypy.yml

```diff
@@ -3,5 +3,5 @@ steps:
       pip install mypy
       echo "Running mypy"
       # configuration is loaded from pyproject.toml
-      mypy . || true
+      mypy tests/common2 || true
```

Comment:
Will the `mypy` check always pass? If yes, then how to block PR merging when check failed?